spec/controllers/v1.0/workflows_controller_spec.rb

```diff
@@ -329,7 +329,7 @@
       before do
         allow(rs_class).to receive(:paginate).and_return([])
         allow(roles_obj).to receive(:roles).and_return([admin_role])
-        patch "#{api_version}/workflows/#{id}", :params => valid_attributes, :headers => default_headers
+        patch "#{api_version}/workflows/#{id}", :params => valid_attributes, :headers => default_headers, :as => :json
```

Comment:
@hsong-rh since this happens by default for specs with type => request do we need to specify the :as => :json since the common gem will do that for you

Reply:
@hsong-rh Maybe https://github.com/ManageIQ/catalog-api/blob/a63a7704f70cb32f78b0bac1974be1038262f6d8/spec/spec_helper.rb#L27 is missing in your spec_helper.rb